railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_0.rb.tt

```diff
@@ -1,8 +1,12 @@
 # Be sure to restart your server when you modify this file.
 #
-# This file contains migration options to ease your Rails 6.0 upgrade.
+# This file contains explicit, new configuration defaults to ease your Rails 6.0
+# upgrade.
 #
-# Once upgraded flip defaults one by one to migrate to the new default.
+# After upgrading Rails, uncomment each of these new defaults one at a time and
+# test your application. If you accept all of the new configuration defaults,
+# you may update the argument to `config.load_defaults` in config/application.rb
+# and delete this file.
```

Comment:
Something feels awkward about the sentence "If you accept all of the new configuration defaults"...

Can you change it to something like this? "Once you've flipped all of the configuration defaults, you can set `config.load_defaults` to `true` in `config/application.rb` and delete this file"?

Reply:
Thanks for the feedback.

Reply:
`config.load_defaults` will barf when passed true: 
https://api.rubyonrails.org/classes/Rails/Application/Configuration.html#method-i-load_defaults

It'll have to be the version you've updated to. In this case you'd do `config.load_defaults 6.0`.

Reply:
@kaspth Yep, see https://github.com/rails/rails/pull/33584/commits/8b09594477efc7d81ce5a8acfb3d2766b9a676ba

Reply:
@matthewd forgot to actually write down that question in my earlier comment 👍

Reply:
@matthewd @kaspth See https://github.com/rails/rails/pull/33584/commits/88edd00004133821da76d561ae39b1f4b5ef4d1b